<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestE2promWidget</class>
 <widget class="QWidget" name="TestE2promWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestE2promWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>命令设置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="SetCmd_lineEdit"/>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QPushButton" name="SetCmdSelectpushButton">
       <property name="text">
        <string>SetCmdSelect</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="StartpushButton">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QSpinBox" name="Cmd_find_spliter_spinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>命令查设置间隔ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>间隔增长量ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QPushButton" name="QueCmdSelectpushButton">
       <property name="text">
        <string>QueCmdSelect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="FindCmd_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>命令查询</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="StoppushButton">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QSpinBox" name="Cmd_find_spliter_Add_spinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>命令查询周期ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="Cmd_find_Timer_spinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>连接成功</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
